### ISSUE TITLE
Support free coordinate fields (ie. not limited to grid)

### ISSUE THREAD

**Issue by deepnight** [deepnight/led#88]

**Comment by Silver-Streak** [comment]
@deepnight Is the intent for this to be for everything, or just initial positions?

I'm unclear if this would solve the issue I'm experiencing, but wanted to check before I opened a new issue: 

Right now if I make a 1x1 grid entity layer, I still cannot position points that match up with the edges of any tile. Only the centers. 

![image](https://user-images.githubusercontent.com/24237149/142058636-0c9393e2-dc1b-4d9e-a6a6-f7665b3b2ebc.png)


As seen in this screenshot here, I can only make the point be below or above the end of a tile. The entity is set to 1x1, and the layer is set to 1px.

Would this fix change this? 

(My goal is to have a way to draw collision polygons/points, and allowing for slopes.)

**Comment by deepnight** [comment]
Yes, the way I manage grid coordinates for Entities is quite broken and needs a deep rework. For the following use cases, you *should* be able use grid or pixel based positioning freely:

 - creating a new Entity (currently works for both)
 - move an existing Entity (currently broken)
 - add a new Point for a field (only grid for now)
 - move an existing Point (only grid for now)

**Comment by Silver-Streak** [comment]
Thanks for the info.

Is that rework planned for 1.0, or later?

(I'm  looking for an easy way to define collision as to help when the GDevelop integration is completed, so I can get another bounty out on importing the points as collision polygons. :D )

**Comment by deepnight** [comment]
Probably after next update (0.9.4) which should be a bug-fix one + save system rework. I'm not 100% sure there will be a 0.10.0, or if I'll jump directly to 1.0.0, as it will depend on how many features I'll be able to push into it.

So I think ETA is about ~1-2 months from now.

**Comment by Silver-Streak** [comment]
Great to hear. I really appreciate the context and info.

**Comment by finscn** [comment]
Slope is very useful .

**Comment by cpuccino** [comment]
Are there any workarounds for this currently? Specifically, coordinate values for "Points".
